client/flutter/lib/pages/onboarding/onboarding_page.dart

```diff
@@ -47,14 +49,20 @@ class _OnboardingPageState extends State<OnboardingPage> {
         controller: _pageController,
         physics: NeverScrollableScrollPhysics(),
         children: <Widget>[
-          LegalLandingPage(onNext: _toNextPage),
+          LegalLandingPage(onNext: _onLegalDone),
           NotificationsPage(onNext: _toNextPage),
           LocationSharingPage(onNext: _onDone),
         ],
       ),
     );
   }
 
+  Future<void> _onLegalDone() async {
+    // Enable auto init so that analytics will work
```

Comment:
Did we test without this?  Did analytics not get sent?

Reply:
Just realized that you were asking a separate question from what I commented below. No, we did not test without this -- I just assumed that FCM autoinit would be necessary for analytics to function.

This means that we need to test:
- with _onLegalDone -- make sure analytics are actually sent after accepting ToS
- without _onLegalDone -- see if analytics are still sent after accepting ToS

Reply:
Please retest this and get it submitted.